nvflare/tool/poc/poc_commands.py

```diff
@@ -753,11 +784,17 @@ def sync_process(service_name, cmd_path):
 
 
 def _run_poc(
-    cmd_type: str, poc_workspace: str, gpu_ids: List[int], service_config: Dict, excluded: list, services_list=None
+    cmd_type: str,
+    poc_workspace: str,
+    gpu_ids: List[int],
+    service_config: Dict,
+    project_config: Dict,
+    excluded: list,
+    services_list=None,
 ):
     if services_list is None:
         services_list = []
-    service_commands = _build_commands(cmd_type, poc_workspace, service_config, excluded, services_list)
+    service_commands = _build_commands(cmd_type, poc_workspace, service_config, project_config, excluded, services_list)
```

Comment:
if the service_commands is already sorted, do we need the time.sleep(2) in line 804 anymore?